.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+<!-- Follow this template for PR's -->
+
+<!--Example Template for adding feature of binary search function
+ -->
+
+<!-- Short description of your changes -->
+
+Adding code snippet of binary search function in algorithms directory
+
+<!-- Add issue no. which corresponds to this PR -->
+
+Closes #no.
+
+## Changes
+
+<!-- Go verbose here & explain in bit detailed way what you have changed -->
+
+-added binary search in algorithms of code snippet
```

Comment:
Bad formatting, full of redundant information, binary search? This should be part of the comment if anything.